reference/robot.md

```diff
@@ -215,110 +284,195 @@ int main() {
+  public Joystick getJoystick();
+  public Keyboard getKeyboard();
+  public LED getLED(String name);
+  public Lidar getLidar(String name);
+  public LightSensor getLightSensor(String name);
+  public Motor getMotor(String name);
+  public Motor getMouse();
+  public Pen getPen(String name);
+  public PositionSensor getPositionSensor(String name);
+  public Radar getRadar(String name);
+  public RangeFinder getRangeFinder(String name);
+  public Receiver getReceiver(String name);
+  public Speaker getSpeaker(String name);
+  public TouchSensor getTouchSensor(String name);
+  // ...
+}
+```
 
-#### `Robot::getAccelerometer`
-#### `Robot::getBrake`
-#### `Robot::getCamera`
-#### `Robot::getCompass`
-#### `Robot::getConnector`
-#### `Robot::getDisplay`
-#### `Robot::getDistanceSensor`
-#### `Robot::getEmitter`
-#### `Robot::getGPS`
-#### `Robot::getGyro`
-#### `Robot::getInertialUnit`
-#### `Robot::getJoystick`
-#### `Robot::getKeyboard`
-#### `Robot::getLED`
-#### `Robot::getLidar`
-#### `Robot::getLightSensor`
-#### `Robot::getMotor`
-#### `Robot::getPen`
-#### `Robot::getPositionSensor`
-#### `Robot::getRadar`
-#### `Robot::getRangeFinder`
-#### `Robot::getReceiver`
-#### `Robot::getSpeaker`
-#### `Robot::getTouchSensor`
-
-[C++](cpp-api.md#cpp_robot) [Java](java-api.md#java_robot) [Python](python-api.md#python_robot)
+%tab-end
 
-```c
-#include <webots/Robot.hpp>
+%tab "MATLAB"
 
-Accelerometer *Robot::getAccelerometer(const std::string &name);
-Brake *Robot::getBrake(const std::string &name);
-Camera *Robot::getCamera(const std::string &name);
-Compass *Robot::getCompass(const std::string &name);
-Connector *Robot::getConnector(const std::string &name);
-Display *Robot::getDisplay(const std::string &name);
-DistanceSensor *Robot::getDistanceSensor(const std::string &name);
-Emitter *Robot::getEmitter(const std::string &name);
-GPS *Robot::getGPS(const std::string &name);
-Gyro *Robot::getGyro(const std::string &name);
-InertialUnit *Robot::getInertialUnit(const std::string &name);
-Joystick *Robot::getJoystick();
-Keyboard *Robot::getKeyboard();
-LED *Robot::getLED(const std::string &name);
-Lidar *Robot::getLidar(const std::string &name);
-LightSensor *Robot::getLightSensor(const std::string &name);
-Motor *Robot::getMotor(const std::string &name);
-Pen *Robot::getPen(const std::string &name);
-PositionSensor *Robot::getPositionSensor(const std::string &name);
-Radar *Robot::getRadar(const std::string &name);
-RangeFinder *Robot::getRangeFinder(const std::string &name);
-Receiver *Robot::getReceiver(const std::string &name);
-Speaker *Robot::getSpeaker(const std::string &name);
-TouchSensor *Robot::getTouchSensor(const std::string &name);
+```matlab
+tag = wb_robot_get_device('name')
 ```
 
+%tab-end
+
+%tab "ROS"
+
+| name | service/topic | data type | data type definition |
+| --- | --- | --- | --- |
+| `/robot/time_step` | `service` | [`webots_ros::set_int`](ros-api.md#common-services) | |
```

Comment:
This function has been copied in the wrong section.
The ROS API doesn't have any method to get the device because it doesn't need it.